cmd/wallet/info.go

```diff
@@ -0,0 +1,90 @@
+package main
+
+import (
+	"time"
+
+	"github.com/pactus-project/pactus/cmd"
+	"github.com/spf13/cobra"
+)
+
+// buildAllInfoCmd builds all sub-commands related to the wallet information.
+func buildAllInfoCmd(parentCmd *cobra.Command) {
+	infoCmd := &cobra.Command{
+		Use:   "info",
+		Short: "retrieving the wallet information.",
+	}
+
+	parentCmd.AddCommand(infoCmd)
+	buildWalletVersionInfoCmd(infoCmd)
```

Comment:
I don't think we need subcommands here. It would be more convenient for the user to get all the wallet information at once.

Reply:
Done 🙂